network-api/networkapi/wagtailcustomization/tests.py

```diff
@@ -7,14 +7,33 @@
 # see https://docs.djangoproject.com/en/3.1/topics/testing/tools/#urlconf-configuration
 @override_settings(STATICFILES_STORAGE="django.contrib.staticfiles.storage.StaticFilesStorage")
 class LocalizedRedirectTests(TestCase):
-    def setUp(self):
+    def test_redirect(self):
+        """Check that we are redirected to the localized version
+        of the homepage when a Redirect object exists.
+        In this example, a Redirect object exists for /test/ -> /
+        so we expect to be redirected to /en/ when we visit /test/
+        """
         redirect = Redirect(old_path="/test", redirect_link="/")
         redirect.save()
-
-    def test_plain_redirect(self):
         response = self.client.get("/test/", follow=True)
         self.assertEqual(response.redirect_chain, [("/", 301), ("/en/", 302)])
 
     def test_localized_redirect(self):
+        """Check that a Redirect with a language code in the old_path
+        and no language code in the redirect_link is handled correctly.
+        We expect to be redirected to /en/final/ when we visit /en/test/.
+        """
+        redirect = Redirect(old_path="/en/test", redirect_link="/final")
+        redirect.save()
+        response = self.client.get("/final/", follow=True)
+        self.assertEqual(response.redirect_chain, [("/en/final/", 302)])
         response = self.client.get("/en/test/", follow=True)
```

Comment:
What do you think about adding empty lines above and below the "action" line? I usually find that helpful for readability.